Jenkinsfile

```diff
@@ -40,4 +40,6 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipeline(['distros' : ['centos7', 'centos8', 'leap15']])
\ No newline at end of file
+packageBuildingPipelineDAOSTest(['distros' : ['el8', 'leap15'],
+                                 'rpmlint_rpms_skip': true,
```

Comment:
this is not our repo and those rpms are not being distributed by us. plus i don't think anyone on our side have the bw to dig in and fix those at this point.
note that those errors have always been there, but it just started failing with this packaging upgrade.

Reply:
This (the packaging -- which is what the linting it checking) **is** our repo.  But even still, not completely papering over the issues would be better so that in such a time as we might distribute this it doesn't happen without us realizing these issues in it because everything is green.

Rather than completely skipping the linting and based on the previous Leap 15 builds, you can add an `hdf5.rpmlintrc` file in the root directory with the contents:

```python
# This line is mandatory to access the configuration functions
from Config import *

addFilter('libtool-wrapper-in-package')
addFilter('binary-or-shlib-defines-rpath')
```

to ignore the *E*rrors.  The warnings should not prevent the pipeline completing successfully but still flags the builds as unstable so that we can (and will always) see that there are still issues with the packaging.

Reply:
ill add that change.. unfortunately now we have to wait another 6hr+ for results..

Reply:
> This (the packaging -- which is what the linting it checking) **is** our repo. But even still, not completely papering over the issues would be better so that in such a time as we might distribute this it doesn't happen without us realizing these issues in it because everything is green.

i just want to be clear, that we will never distribute this. we are not responsible ourselves for the HDF5 product.